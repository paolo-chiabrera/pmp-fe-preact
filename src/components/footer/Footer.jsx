import { h } from 'preact';

export default function Footer() {
  return <footer />;
}
