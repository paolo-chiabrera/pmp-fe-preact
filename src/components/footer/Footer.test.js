import { h } from 'preact';

import render from 'preact-render-to-string';

import Footer from './Footer.jsx';

describe('Given a Footer component', () => {
  it('should render correctly', () => {
    const tree = render(<Footer />);

    expect(tree).toMatchSnapshot();
  });
});
