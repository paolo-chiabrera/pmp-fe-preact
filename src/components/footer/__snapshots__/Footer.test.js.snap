// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Footer component should render correctly 1`] = `"<footer></footer>"`;
